UPGRADE-2.1.md

```diff
@@ -0,0 +1,137 @@
+
+1. Download it from the website [PIM community standard](http://www.akeneo.com/download/) and extract:
+
+    ```bash
+    wget http://download.akeneo.com/pim-community-standard-v1.7-latest.tar.gz
+    tar -zxf pim-community-standard-v1.7-latest.tar.gz
+    cd pim-community-standard/
+    ```
+
+2. Copy the following files to your PIM installation:
+
+    ```bash
+    export PIM_DIR=/path/to/your/pim/installation
+ 
+    mv $PIM_DIR/app/config/config.yml $PIM_DIR/app/config/config.yml.bak
+    cp app/config/config.yml $PIM_DIR/app/config
+ 
+    mv $PIM_DIR/app/config/pim_parameters.yml $PIM_DIR/app/config/pim_parameters.yml.bak
+    cp app/config/pim_parameters.yml $PIM_DIR/app/config
+
+    mv $PIM_DIR/composer.json $PIM_DIR/composer.json.bak
+    cp composer.json $PIM_DIR/
+    ```
+
+3. Remove your old upgrades folder:
+
+    ```bash
+    rm -rf $PIM_DIR/upgrades/schema
+    ```
+
+4. Update your dependencies:
+
+    * [Optional] If you had added dependencies to your project, you will need to do it again in your `composer.json`.
+    You can display the differences of your previous composer.json in `$PIM_DIR/composer.json.bak`.
+    
+    ```JSON
+    "require": {
+       "your/dependency": "version",
+       "your/other-dependency": "version",
+    }
+    ```
+
+   * Then run the command to update your dependencies:
+
+    ```bash
+    cd $PIM_DIR
+    php -d memory_limit=3G ../composer.phar update
+    ```
+    
+    This step will copy the upgrades folder from `pim-community-dev/` to your Pim project root in order to migrate.
+    If you have custom code in your project, this step may raise errors in the "post-script" command.
+    In this case, go to the chapter "Migrate your custom code" before running the database migration.
+ 
+5. Migrate your database:
+ 
+    ```bash
+    rm -rf var/cache
+    bin/console doctrine:migration:migrate --env=prod
+    ```
+
+6. Create the missing internal job instances:
+
+    Depending on the version of your current PIM, you'll have to execute these commands to create new internal jobs.
+
+    * From 2.0.4 and below
+    
+    ```bash
+    bin/console akeneo:batch:create-job "internal" "compute_completeness_of_products_family" "compute_completeness_of_products_family" "compute_completeness_of_products_family" '{"family_code":"null"}' "compute completeness of products family" --env=prod
+    ```
+    
+    * From 2.0.5 and below
+    
+    ```bash
+    bin/console akeneo:batch:create-job "internal" "compute_family_variant_structure_changes" "compute_family_variant_structure_changes" "compute_family_variant_structure_changes" '{"family_variant_codes":["null"]}' "Compute family variant structure changes" --env=prod
+    ```
+    
+    * From 2.0.7 and below
+    
+    ```bash
+    bin/console akeneo:batch:create-job internal add_to_existing_product_model mass_edit add_to_existing_product_model '{}' 'Add products to an existing product model' --env=prod
+    ```
+    
+    * From any version
```

Comment:
So, from 2.0.0? Maybe put at the start instead, to be ordered.

Reply:
It's sorted from the smallest version to the greater. The commands below "From any version" are those that have been added in 2.1